frontend/src/constants/styleConstants.ts

```diff
@@ -1,13 +1,13 @@
 const COLORS = {
```

Comment:
Create a theme file in assets folder. There can be colors, font size, media queries, etc. 

For example: [https://github.com/Skolaczk/Audiophile/blob/main/src/assets/styles/theme.ts](https://github.com/Skolaczk/Audiophile/blob/main/src/assets/styles/theme.ts)

Reply:
done